api/v1/db/oracledb/games-dao.js

```diff
@@ -1,4 +1,5 @@
 const appRoot = require('app-root-path');
+const oracledb = require('oracledb');
 const _ = require('lodash');
```

Comment:
The package import order should follow the [style guild](https://github.com/osu-mist/style-guides/blob/master/JavaScript.md#requiring-modules):

> Each line within groups 1-3 should be ordered alphabetically by the first alphanumeric character in the name of the package/file in the statement.

Reply:
Fixed import order in games-dao and developers-dao.